en/mulesoft/modules/ROOT/pages/ms_composer_invocable_flows.adoc

```diff
@@ -0,0 +1,263 @@
+* Retrieves an order status from NetSuite and posts a message in Slack.
+* Gets real-time credit scores when a user goes through a screen flow.
+* Retrieves an order status from NetSuite when a user enters an order ID in the screen flow.
+
+To invoke a Composer flow from Flow Builder, you must:
+
+. <<set-up-salesforce-for-invocable-flows,Set up Salesforce for invocable flows>>.
+. <<set-up-flow-builder-for-invocable-flows,Set up Flow Builder for invocable flows>>.
+
+[[set-up-salesforce-for-invocable-flows]]
+==== Set Up Salesforce for Invocable Flows
+
+After creating your Composer flow, set up your Salesforce org to invoke flows in Composer: 
+
+. Access your Salesforce org. 
+. Go to *Setup* > *MuleSoft* > *Services* and click *Log In* to securely connect Anypoint Platform with your Salesforce org and grant access.
+. Create a new set of credentials that use the same credentials that you use for Composer. Create an external credential and a named credential. If your invocable Composer flow does not contain input fields, you must add a new `Content-Length` custom header that has a value of `0` in the external credential and named credential to avoid an HTTP 411 response. 
++
+To create an external credential and a named credential, you must:
+
+.. https://help.salesforce.com/s/articleView?id=sf.nc_create_edit_custom_auth_ext_cred.htm&type=5[Create and Edit a Custom Authentication External Credential].
+.. https://help.salesforce.com/s/articleView?id=sf.nc_enable_ext_cred_principal.htm&type=5[Enable External Credential Principals].
+.. https://help.salesforce.com/s/articleView?id=sf.nc_custom_headers_basic_auth.htm&type=5[Use Basic Authentication with Named Credentials].  
+.. https://help.salesforce.com/s/articleView?id=sf.nc_create_edit_named_credential.htm&type=5[Create and Edit a Named Credential]. 
+
+. Register your external service by copying the API specification from the Composer flow. For more information, refer to https://help.salesforce.com/s/articleView?id=sf.external_services_register_json.htm&type=5[Paste the JSON-formatted Schema Into Your Browser]. 
+
+[[set-up-flow-builder-for-invocable-flows]]
+==== Set Up Flow Builder for Invocable Flows 
+
+After setting up Salesforce for invocable flows, you can call the Composer flow from Flow Builder:
+
+. Access your Salesforce org.
+. Go to *Setup* > *Process Automation* > *Flows*.
+. Click *New Flow* to create a new flow in Flow Builder. 
+. Select the type of flow you want to create, for example, a *Record-Triggered Flow*, and then click *Create*.
+. Configure the start of the flow. If you create a record-triggered flow, you must select the *Run Asynchronously* checkbox to avoid errors.
+. Set up an invocable action. Click *+* to add a new element and then click *Action*.
+. In *Filter By*, select *Type* and click *External Service*. 
+. Select the action of the invocable flow from the external service.
+. Enter a *Label* and *API Name*. 
```

Comment:
Enter a label and API name
or
Enter a value for the *Label* and *API Name* field.